### PR TITLE
Feature/prep5 6 rel

### PR DESCRIPTION
**Pull request by TroyWalshProf**
Release prep
Addresses issue #40

### CODE REVIEW

File: Extensions/CoreTemplates/Magenic.Maqs.Appium.Template/Tests/appsettings.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
     "Timeout": "10000",
     "Log": "OnFail",
     "LogLevel": "INFORMATION",
-    "LogType": "TXT"
+    "LogType": "TXT",
+    "UseFirstChanceHandler": "YES"
```
Comment (FermJacob) [inline Extensions/CoreTemplates/Magenic.Maqs.Appium.Template/Tests/appsettings.json]: Do we need this defaulted?

File: Extensions/CoreTemplates/Magenic.Maqs.Base.Template/Tests/appsettings.json [hunk 2]
```diff
@@ -4,6 +4,7 @@
     "Timeout": "10000",
     "Log": "OnFail",
     "LogLevel": "INFORMATION",
-    "LogType": "TXT"
+    "LogType": "TXT",
+    "UseFirstChanceHandler": "YES"
```
Comment (FermJacob) [inline Extensions/CoreTemplates/Magenic.Maqs.Base.Template/Tests/appsettings.json]: Same.  It defaults to YES and is documented

File: Extensions/MaqsSpecFlowExtension/ReleaseNotes.txt [hunk 3]
```diff
@@ -1,49 +1,58 @@
 MAQS SpecFlow ExtensionRelease Notes:
+5.6.0 - 2019/10/10
+    Bug fixes and minor enhancements
+    - Logging improvements
+    Package update
+    - Add support for .Net Core 3
+    - Mass dependency updates
+    Template updates
+    - Mass dependency updates
+
```
Comment (FermJacob) [inline Extensions/MaqsSpecFlowExtension/ReleaseNotes.txt]: Do we want to add https://github.com/Magenic/MAQS/issues/31

File: Framework/CoreUnitTests/CoreUnitTests.csproj [hunk 4]
```diff
@@ -1,20 +1,20 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
```
Comment (FermJacob) [inline Framework/CoreUnitTests/CoreUnitTests.csproj]: Do we want to target Core 2.0 and 3.0 here?

File: Framework/NugetBuild/package.nuspec [hunk 5]
```diff
@@ -13,6 +13,15 @@
...
     <summary>Magenic's automation quick start framework. MAQS is a modular test automation framework that can be used as the base for your automation project or individual pieces can be used to enhance existing frameworks. Utilizing project templates, NuGet, and utility libraries we are able to have a functioning framework up and running in minutes, almost entirely removing on the initial time investment on implementing an automation solution.</summary>
     <releaseNotes>
+      5.6.0 - 2019/10/10
+      Bug fixes and minor enhancements
+      - Logging improvements
+      Package update
+      - Add support for .Net Core 3
+      - Mass dependency updates
+      Template updates
+      - Mass dependency updates
```
Comment (FermJacob) [inline Framework/NugetBuild/package.nuspec]: Want to add information about https://github.com/Magenic/MAQS/issues/31

File: Framework/NugetBuild/packageSpecFlow.nuspec [hunk 6]
```diff
@@ -13,54 +13,63 @@
...
-    5.5.2 - 2019/9/5
+      5.6.0 - 2019/10/10
+      Bug fixes and minor enhancements
+      - Logging improvements
+      Package update
+      - Add support for .Net Core 3
+      - Mass dependency updates
+      Template updates
+      - Mass dependency updates
+
```
Comment (FermJacob) [inline Framework/NugetBuild/packageSpecFlow.nuspec]: https://github.com/Magenic/MAQS/issues/31

File: TemplateUpdate.ps1 [hunk 7]
```diff
@@ -37,6 +37,11 @@
   Creation Date:  3/28/2019
   Purpose/Change: Update mongo as well
   
+  Version:        6.0
```
Comment (FermJacob) [inline TemplateUpdate.ps1]: Assuming you tested this and this PR is the outcome?

File: Framework/BaseMongoTest/BaseMongoTest.csproj [hunk 8]
```diff
@@ -1,8 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net471;netstandard2.0</TargetFrameworks>
-    <Version>5.5.1</Version>
+    <TargetFrameworks>net471;netstandard2.0;netstandard2.1</TargetFrameworks>
+    <Version>5.6.0</Version>
```
Comment (FermJacob) [inline Framework/BaseMongoTest/BaseMongoTest.csproj]: Are we missing
<AssemblyVersion>5.6.0</AssemblyVersion>
    <FileVersion>5.6.0</FileVersion>

File: Framework/BaseEmailTest/BaseEmailTest.csproj [hunk 9]
```diff
@@ -1,14 +1,14 @@
...
-    <TargetFrameworks>net471;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net471;netstandard2.0;netstandard2.1</TargetFrameworks>
     <AssemblyName>Magenic.Maqs.BaseEmailTest</AssemblyName>
     <RootNamespace>Magenic.Maqs.BaseEmailTest</RootNamespace>
     <PackageId>Magenic.Maqs.BaseEmailTest</PackageId>
     <Authors>Magenic.Maqs.BaseEmailTest</Authors>
     <Company>Magenic.Maqs.BaseEmailTest</Company>
     <Product>Magenic.Maqs.BaseEmailTest</Product>
-    <Version>5.5.1</Version>
+    <Version>5.6.0</Version>
```
Comment (FermJacob) [inline Framework/BaseEmailTest/BaseEmailTest.csproj]: <AssemblyVersion>5.6.0</AssemblyVersion>
    <FileVersion>5.6.0</FileVersion>

File: Framework/BaseDatabaseTest/BaseDatabaseTest.csproj [hunk 10]
```diff
@@ -1,10 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net471;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net471;netstandard2.0;netstandard2.1</TargetFrameworks>
     <AssemblyName>Magenic.Maqs.BaseDatabaseTest</AssemblyName>
     <RootNamespace>Magenic.Maqs.BaseDatabaseTest</RootNamespace>
-    <Version>5.5.1</Version>
```
Comment (FermJacob) [inline Framework/BaseDatabaseTest/BaseDatabaseTest.csproj]:
```
<AssemblyVersion>5.6.0</AssemblyVersion>
    <FileVersion>5.6.0</FileVersion>
```

File: Framework/BaseAppiumTest/BaseAppiumTest.csproj [hunk 11]
```diff
@@ -1,10 +1,10 @@
 ﻿  <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net471;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net471;netstandard2.0;netstandard2.1</TargetFrameworks>
     <AssemblyName>Magenic.Maqs.BaseAppiumTest</AssemblyName>
     <RootNamespace>Magenic.Maqs.BaseAppiumTest</RootNamespace>
-    <Version>5.5.1</Version>
+    <Version>5.6.0</Version>
```
Comment (FermJacob) [inline Framework/BaseAppiumTest/BaseAppiumTest.csproj]:
```
<AssemblyVersion>5.6.0</AssemblyVersion>
    <FileVersion>5.6.0</FileVersion>
```

File: Extensions/VisualStudioQatExtensionOss/ReleaseNotes.txt [hunk 12]
```diff
@@ -1,4 +1,13 @@
 MAQS Framework Release Notes:
+5.6.0 - 2019/10/10
+    Bug fixes and minor enhancements
+    - Logging improvements
+    Package update
+    - Add support for .Net Core 3
+    - Mass dependency updates
+    Template updates
+    - Mass dependency updates
```
Comment (FermJacob) [inline Extensions/VisualStudioQatExtensionOss/ReleaseNotes.txt]: https://github.com/Magenic/MAQS/issues/31

File: Extensions/VisualStudioQatExtension/ReleaseNotes.txt [hunk 13]
```diff
@@ -1,4 +1,13 @@
 Maqs Framework Release Notes:
+5.6.0 - 2019/10/10
+    Bug fixes and minor enhancements
+    - Logging improvements
+    Package update
+    - Add support for .Net Core 3
+    - Mass dependency updates
+    Template updates
+    - Mass dependency updates
+
```
Comment (FermJacob) [inline Extensions/VisualStudioQatExtension/ReleaseNotes.txt]: https://github.com/Magenic/MAQS/issues/31

File: Extensions/CoreTemplates/Magenic.Maqs.Appium.Template/Tests/appsettings.json (hunk 1 above)
Comment (TroyWalshProf) [reply Extensions/CoreTemplates/Magenic.Maqs.Appium.Template/Tests/appsettings.json]: Need no, want yes.
This is in lieu of adding comments to JSON.
*This makes it easier for us to tell folks that are hitting the cross thread logging issues with MSTest to just turn this off.

File: Extensions/CoreTemplates/Magenic.Maqs.Base.Template/Tests/appsettings.json (hunk 2 above)
Comment (TroyWalshProf) [reply Extensions/CoreTemplates/Magenic.Maqs.Base.Template/Tests/appsettings.json]: Ditto

File: Framework/CoreUnitTests/CoreUnitTests.csproj (hunk 4 above)
Comment (TroyWalshProf) [reply Framework/CoreUnitTests/CoreUnitTests.csproj]: Yep

File: Extensions/MaqsSpecFlowExtension/ReleaseNotes.txt (hunk 3 above)
Comment (TroyWalshProf) [reply Extensions/MaqsSpecFlowExtension/ReleaseNotes.txt]: They are for two different sections and it is true for both

File: Framework/NugetBuild/package.nuspec (hunk 5 above)
Comment (TroyWalshProf) [reply Framework/NugetBuild/package.nuspec]: They are for two different sections and it is true for both

File: Framework/BaseEmailTest/BaseEmailTest.csproj (hunk 9 above)
Comment (TroyWalshProf) [reply Framework/BaseEmailTest/BaseEmailTest.csproj]: Yes, that is our standard convention

File: Framework/BaseAppiumTest/BaseAppiumTest.csproj (hunk 11 above)
Comment (TroyWalshProf) [reply Framework/BaseAppiumTest/BaseAppiumTest.csproj]: Not sure what you are asking

File: Extensions/VisualStudioQatExtensionOss/ReleaseNotes.txt (hunk 12 above)
Comment (TroyWalshProf) [reply Extensions/VisualStudioQatExtensionOss/ReleaseNotes.txt]: Ditto

File: Extensions/VisualStudioQatExtension/ReleaseNotes.txt (hunk 13 above)
Comment (TroyWalshProf) [reply Extensions/VisualStudioQatExtension/ReleaseNotes.txt]: ditto

**Pull request merged**